.env.sample

```diff
@@ -0,0 +1,38 @@
+# Private key of the account Hardhat will use
+PRIVATE_KEY=
+
+# ENV vars for MongoDB, for local development you can use below
+MONGO_DB_URI="mongodb://localhost:27018"
+MONGO_DB_NAME="zns-campaign"
+MONGO_DB_VERSION="7.0"
+MONGO_DB_CLIENT_OPTS="<optional>"
+
+# ENV vars for Logger
+LOG_LEVEL="info" | "debug"
+SILENT_LOGGER="false" | "true"
+
+# Boolean value for if we deploy the mock
+# true = we deploy the mock
+# false = we use a hard coded address and pull data from chain
+MOCK_MEOW_TOKEN=
+
+# Environment variables to create an entirely custom config when `env_level` above is not dev
+MAX_PRICE=
+MIN_PRICE=
+MAX_LENGTH=
+BASE_LENGTH=
+
+DECIMALS=
+PRECISION=
+REG_FEE_PERCENT=
+ROYALTY_FRACTION=
+
+TOKEN_NAME=
+TOKEN_SYMBOL=
+ROYALTY_RECEIVER=
+ZERO_VAULT_ADDRESS=
+STAKING_TOKEN_ADDRESS=
+
+# comma separated list of address e.g. "0x123,0x456,..."
```

Comment:
plz specify that they should be added as base64 string

Reply:
Agreed, this change is already on my other branch [deploy/validate-env](https://github.com/zer0-os/zNS/pull/73/files#diff-088d9f35d23a4347d221d71dd49b02b95001dff4abe637a40fe0bc04d502049c), after merging my (`deploy/script`)[https://github.com/zer0-os/zNS/pull/74] that PR can be merged into this branch, which resolves the issue